plugins/module_utils/utils.py

```diff
@@ -0,0 +1,56 @@
+from __future__ import absolute_import, division, print_function
+
+import os
+import json
+from typing import List, Optional, Union, cast
+
+from ansible_collections.cloud.terraform.plugins.module_utils.types import (
+    TJsonBareValue,
+    TJsonObject,
+    AnsibleRunCommandType,
+)
+from ansible_collections.cloud.terraform.plugins.module_utils.errors import TerraformWarning, TerraformError
+
+
+__metaclass__ = type
+
+
+def get_state_args(state_file: Optional[str]) -> List[str]:
+    if state_file is not None:
+        if os.path.exists(state_file):
+            return ["-state", state_file]
+        else:
+            raise TerraformError('Could not find state_file "{0}", check the path and try again.'.format(state_file))
```

Comment:
```suggestion
        if not os.path.exists(state_file):
            raise TerraformError('Could not find state_file "{0}", check the path and try again.'.format(state_file))
        return ["-state", state_file] 
```
just for code complexity, nothing mandatory